Lib/Src/CUartCom.cpp

```diff
@@ -275,7 +275,7 @@ etl::string<MAX_STRING_SIZE> CUartCom::getString()
  * @brief Check if the queue has any commands to read
  * @return True if the queue is not empty
  */
-bool CUartCom::isCommandAvailable()
+bool CUartCom::isDataAvailable()
```

Comment:
Does doxygen header need updating? It still says "command", while the method name is `isDataAvailable()`.
Should probably also define what it means that data is available.

Reply:
Fixed.